### PR TITLE
fw_pos_control_l1 rename files

### PR DESCRIPTION
**Pull request by dagar**
These giant cpp files are hard to navigate and cause problems for some IDEs. There's no active work in this module so it seemed like a good time.

I made the change such that git blame is able to follow fw_pos_control_l1.cpp -> FixedwingPositionControl.cpp.

### CODE REVIEW

**Comment by dagar** [comment]
@Stifael Can you do a quick review? This is FW, but just a style cleanup. Going forward I want to do a better job breaking up the massive cpp files with separate headers and consistent naming that matches the actual class.

**Comment by LorenzMeier** [comment]
@bresch @tumbili Could you please both have a pass at this?

File: src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp [hunk 1]
```diff
@@ -851,38 +405,17 @@ FixedwingPositionControl::get_demanded_airspeed()
 float
 FixedwingPositionControl::calculate_target_airspeed(float airspeed_demand)
 {
-	float airspeed;
-
-	if (_airspeed_valid) {
```
Comment (Stifael) [inline src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp]: this _airspeed_valid check is not needed?

Comment (dagar) [inline src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp]: If you look at the old code it's only used in this function to calculate airspeed_error, which is never used. One of the reasons I started doing this is the controller had a lot of code that looked real, but was actually doing nothing. I've previously done several passes of deleting.

**Comment by bresch** [comment]
@dagar  It's a good step in the right direction! Those "everything-in-cpp" files are way to big and messy.
@LorenzMeier Everything looks fine for me

**Pull request merged**